filters/all.js

```diff
@@ -1,55 +1,84 @@
-const _ = require('lodash');
-const Markdown = require('markdown-it');
-const OpenAPISampler = require('openapi-sampler');
-
-module.exports = ({ Nunjucks }) => {
-  Nunjucks.addFilter('split', (string, separator) => {
-    if (typeof string !== 'string') return string;
-    const regex = new RegExp(separator, 'g');
-    return string.split(regex);
-  });
-
-  Nunjucks.addFilter('firstKey', (obj) => {
-    if (!obj) return '';
-    return Object.keys(obj)[0];
-  });
-
-  Nunjucks.addFilter('isExpandable', (obj) => {
-    if (
-      obj.type() === 'object' ||
-      obj.type() === 'array' ||
-      (obj.oneOf() && obj.oneOf().length) ||
-      (obj.anyOf() && obj.anyOf().length) ||
-      (obj.allOf() && obj.allOf().length) ||
-      obj.items() ||
-      obj.additionalItems() ||
-      (obj.properties() && Object.keys(obj.properties()).length) ||
-      obj.additionalProperties() ||
-      (obj.extensions() && Object.keys(obj.extensions()).filter(e => !e.startsWith('x-parser-')).length) ||
-      obj.patternProperties()
-    ) return true;
-
-    return false;
-  });
-
-  /**
-   * Check if there is a channel which does not have one of the tags specified.
-   */
-  Nunjucks.addFilter('containTags', (object, tagsToCheck) => {
-    if (!object) {
-      throw new Error("object for containsTag was not provided?");
-    }
-
-    if (!tagsToCheck) {
-      throw new Error("tagsToCheck for containsTag was not provided?");
+const filter = module.exports;
+
+function isExpandable(obj) {
+  if (
+    obj.type() === 'object' ||
+    obj.type() === 'array' ||
+    (obj.oneOf() && obj.oneOf().length) ||
+    (obj.anyOf() && obj.anyOf().length) ||
+    (obj.allOf() && obj.allOf().length) ||
+    obj.items() ||
+    obj.additionalItems() ||
+    (obj.properties() && Object.keys(obj.properties()).length) ||
+    obj.additionalProperties() ||
+    (obj.extensions() && Object.keys(obj.extensions()).filter(e => !e.startsWith('x-parser-')).length) ||
+    obj.patternProperties()
+  ) return true;
+
+  return false;
+}
+filter.isExpandable = isExpandable;
```

Comment:
Same as in the other PR. We can probably rewrite this as follows:

```suggestion
filter.isExpandable = function(obj) {
  if (
    obj.type() === 'object' ||
    obj.type() === 'array' ||
    (obj.oneOf() && obj.oneOf().length) ||
    (obj.anyOf() && obj.anyOf().length) ||
    (obj.allOf() && obj.allOf().length) ||
    obj.items() ||
    obj.additionalItems() ||
    (obj.properties() && Object.keys(obj.properties()).length) ||
    obj.additionalProperties() ||
    (obj.extensions() && Object.keys(obj.extensions()).filter(e => !e.startsWith('x-parser-')).length) ||
    obj.patternProperties()
  ) return true;

  return false;
};
```

Reply:
here I can do it this way, unless we would like to publish API.md for template filters too?

Reply:
Yeah, API.md for filters would be great too. Leave it as is so we can add docs later 👍